src/space-commander/space-commander-main.cpp

```diff
@@ -25,6 +27,7 @@ const char ERROR_EXECUTING_COMMAND = '2';
 // Declarations
 static void out_of_memory_handler();
 static int perform(int bytes);
+static void validate();
```

Comment:
Let's have validate return an integer representing errorcodes [(link)](https://github.com/spaceconcordia/space-lib/blob/master/include/SpaceDecl.h#L118-126) and have the commands calling validate return them to the ground